torch/nn/modules/module.py

```diff
@@ -1426,6 +1429,34 @@ def _register_load_state_dict_pre_hook(self, hook, with_module=False):
         self._load_state_dict_pre_hooks[handle.id] = hook
         return handle
 
+    def register_load_state_dict_post_hook(self, hook):
+        r"""Registers a post hook to be run after module's ``load_state_dict``
+        is called. The hook will be called everytime after :func:`load_state_dict`
+        has loaded a ``state_dict`` into the ``self`` module.
+        It should have the following signature::
+            hook(module, incompatible_keys) -> None
+        The ``module`` argument is the current module that this hook is registered
+        on, and the ``incompatible_keys`` argument is a ``NamedTuple`` consisting
+        of attributes ``missing_keys`` and ``unexpected_keys``. ``missing_keys``
+        is a ``list`` of ``str`` containing the missing keys and
+        ``unexpected_keys`` is a ``list`` of ``str`` containing the unexpected keys.
+        If the hook wishes to modify the ``NamedTuple`` ``missing_keys``
```

Comment:
Not sure about that sentence.
It can just be: "The given incompatible_keys can be modified inplace if need". (this is a named tuple so people won't be able to re-assign the elements anyways.